service/src/grandpa_support.rs

```diff
@@ -244,13 +244,21 @@ mod tests {
 
 	#[test]
 	fn grandpa_pause_voting_rule_works() {
+		env_logger::init();
```

Comment:
```suggestion
		let _ = env_logger::try_init();
```

Never call `init()` in a test.